type.py

```diff
@@ -63,9 +64,29 @@ class PreprocessConfig:
     input_col: str
     label_col: str
 
+    def get_configs(self):
+        return vars(self)
+
 
 @dataclass
 class PytorchConfig(BaseConfig):
     hidden_size: int
     output_size: int
     val_size: float
+
+
+@dataclass
+class RunConfig:
+    run_name: str  # Get's appended as a prefix before the pipeline name
+    train: bool  # Weather the run should do training
+    dataset: Dataset
```

Comment:
this is pd.DataFrame or Union[TestDataset, TrainDataset]

Reply:
I don't have yet TestDataset (its in the other PR) so I'll ad pd.Dataframe